wrappers/src/table_cs.cpp

```diff
@@ -198,130 +192,145 @@ REALM_EXPORT size_t table_get_nullable_timestamp_milliseconds(const Table* table
         table_ptr->set_binary(column_ndx, row_ndx, BinaryData(value, value_len));
     });
 }
 
-REALM_EXPORT void table_set_timestamp_milliseconds(Table* table_ptr, size_t column_ndx, size_t row_ndx, int64_t value)
+REALM_EXPORT void table_set_timestamp_milliseconds(Table* table_ptr, size_t column_ndx, size_t row_ndx, int64_t value, NativeException::Marshallable& ex)
 {
-    return handle_errors([&]() {
+    return handle_errors(ex, [&]() {
         table_ptr->set_timestamp(column_ndx, row_ndx, from_milliseconds(value));
     });
 }
 
-REALM_EXPORT Query* table_where(Table* table_ptr)
+REALM_EXPORT Query* table_where(Table* table_ptr, NativeException::Marshallable& ex)
 {
-    return handle_errors([&]() {
+    return handle_errors(ex, [&]() {
       return new Query(table_ptr->where());
     });
 }
 
-REALM_EXPORT int64_t table_count_all(Table* table_ptr)
+REALM_EXPORT int64_t table_count_all(Table* table_ptr, NativeException::Marshallable& ex)
 {
-    return handle_errors([&]() {
+    return handle_errors(ex, [&]() {
       return table_ptr->size();
     });
 }
 
-REALM_EXPORT size_t table_get_column_index(Table* table_ptr, uint16_t *  column_name, size_t column_name_len)
+REALM_EXPORT size_t table_get_column_index(Table* table_ptr, uint16_t *  column_name, size_t column_name_len, NativeException::Marshallable& ex)
 {
-    return handle_errors([&]() {
+    return handle_errors(ex, [&]() {
         Utf16StringAccessor str = Utf16StringAccessor(column_name, column_name_len);
         return table_ptr->get_column_index(str);
     });
 }
 
-REALM_EXPORT size_t tableview_get_column_index(TableView* tableView_ptr, uint16_t *  column_name, size_t column_name_len)
+REALM_EXPORT size_t tableview_get_column_index(TableView* tableView_ptr, uint16_t *  column_name, size_t column_name_len, NativeException::Marshallable& ex)
 {
-    return handle_errors([&]() {
+    return handle_errors(ex, [&]() {
         Utf16StringAccessor str = Utf16StringAccessor(column_name, column_name_len);
         return tableView_ptr->get_column_index(str);
     });
 }
 
-REALM_EXPORT void table_remove_row(Table* table_ptr, Row* row_ptr)
+REALM_EXPORT void table_remove_row(Table* table_ptr, Row* row_ptr, NativeException::Marshallable& ex)
 {
-    handle_errors([&]() {
+    handle_errors(ex, [&]() {
         table_ptr->move_last_over(row_ptr->get_index());
     });
 }
 
+REALM_EXPORT Results* table_create_results(Table* table_ptr, SharedRealm* realm, ObjectSchema* object_schema, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        return new Results(*realm, *object_schema, *table_ptr);
+    });
+}
+
+REALM_EXPORT Results* table_create_sorted_results(Table* table_ptr, SharedRealm* realm, ObjectSchema* object_schema, SortOrderWrapper* sortorder_ptr, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        return new Results(*realm, *object_schema, Query(table_ptr->where()), sortorder_ptr->sort_order);
```

Comment:
Is the explicit call to the `Query` constructor here necessary? `Table::where` returns a `Query` so it looks redundant to me.